src/app/lib/analyticsUtils/index.js

```diff
@@ -125,6 +125,19 @@ export const getReferrer = (platform, origin, previousPath) => {
   return null;
 };
 
+export const getAtUserId = platform => {
+  if (platform === 'amp') {
+    return null;
+  }
+
+  if (onClient()) {
```

Comment:
Can combine these `if`s

```suggestion
  if (onClient() && platform !== 'amp') {
```
Technically, `onClient()` is never true on AMP pages, as we dont run client JS, but nice to be explicty